fc-community-contracts/src/FactChainNFT.sol

```diff
@@ -5,29 +5,45 @@ import {ERC721URIStorage} from "openzeppelin-contracts/contracts/token/ERC721/ex
 import {ERC721} from "openzeppelin-contracts/contracts/token/ERC721/ERC721.sol";
 import {Ownable} from "./utils/Ownable.sol";
 
+contract FactChainSFT {
+    function mint(address[] memory raters, string memory ipfsHash, uint256 id) public returns (uint256) {}
+}
+
 /// @title FactChainers Community Notes NFT
 /// @author Pierre HAY
 /// @notice
 /// @dev
 contract FactChainNFT is Ownable, ERC721URIStorage {
+    FactChainSFT factsLithography;
```

Comment:
maybe add a setter to be able to redeploy easily

Reply:
done